### ISSUE TITLE
Perform Train, Test and Split and Use k-nearest neighbors model #4

### ISSUE THREAD

**Issue by hasabnistejas** [CSI-CATT-DMCE/Edible_Mushroom_Prediction#4]
- For Train, Test and Split parameter test_size should be 0.15 and random_state=101. Here's a reference for the method [ref](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html) and result should be stored in the variable __X_train, X_test, y_train, y_test__ .

- Apply k-nearest neighbours algorithm for classification with the value of k to be matching according to the dataset.

- Here's a reference for the algorithm using SciKit for [kNN algorithm](https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html#sklearn.neighbors.KNeighborsClassifier.kneighbors)

- Please use the variable named _'knnmodel'_ for storing the model.

Note: Please mention the accuracy percentage while submitting pull requests by running the evaluation code.
